src/app/services/utils.service.ts

```diff
@@ -10,21 +10,29 @@ declare var window: any;
 // contact number format should be consistent throughout the app (GoMobile & Setting)
 export class ContactNumberFormat {
```

Comment:
Since we have a separate component for contact number now, there's no need to put this in utils.service anymore